zerver/openapi/zulip.yaml

```diff
@@ -4841,6 +4841,19 @@ paths:
 
                                         **Changes**: Before Zulip 6.0 (feature level 138), no limit was
                                         represented using the special value `0`.
+                                    moderation_request_channel_id:
+                                      type: integer
+                                      description: |
+                                        The ID of the private channel to which messages flagged for moderation
+                                        requests by users are sent. Moderators can use this channel to review and
+                                        act on reported content.
```

Comment:
"flagged by users for moderation" might be clearer wording.

Reply:
Updated to:
```yaml

  The ID of the private channel to which messages flagged by users for
  moderation are sent. Moderators can use this channel to review and
  act on reported content.
```